<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ShortcutConfigBase</class>
 <widget class="QDialog" name="ShortcutConfigBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configure Keyboard Shortcuts</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox3" >
     <property name="title" >
      <string/>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>3</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="cgListView" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>105</width>
          <height>200</height>
         </size>
        </property>
        <property name="indentation" >
         <number>0</number>
        </property>
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
        <property name="columnCount" >
         <number>1</number>
        </property>
        <property name="sortingEnabled" >
         <bool>true</bool>
        </property>
        <column>
         <property name="text" >
          <string>Category</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="scListView" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>170</width>
          <height>230</height>
         </size>
        </property>
        <property name="indentation" >
         <number>0</number>
        </property>
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
        <property name="columnCount" >
         <number>2</number>
        </property>
        <property name="sortingEnabled" >
         <bool>true</bool>
        </property>
        <column>
         <property name="text" >
          <string>Description</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Shortcut</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>150</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Clear</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="defineButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Define</string>
       </property>
       <property name="shortcut" >
        <string>Alt+D</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>30</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton" >
       <property name="text" >
        <string>&amp;Apply</string>
       </property>
       <property name="shortcut" >
        <string>Alt+A</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
